src/yaml-support/tkn-yaml.ts

```diff
@@ -93,7 +99,7 @@ export class TektonYaml {
     if (rootMap) {
       const apiVersion = getYamlMappingValue(rootMap, 'apiVersion');
       const kind = getYamlMappingValue(rootMap, 'kind');
-      if (apiVersion && apiVersion.startsWith(TEKTON_API)) {
+      if (apiVersion && apiVersion.startsWith(TEKTON_API) || apiVersion.startsWith(TRIGGER_API)) {
```

Comment:
It is better to replace this with 
```typescript
if(apiVersion?.startsWith(TEKTON_API) || apiVersion?.startsWith(TRIGGER_API))
```
Just hawing `|| apiVersion.startsWith(TRIGGER_API)` may throw error in case if `apiVersion` is `undefined`